tests/integration/models/reddit/test_comment.py

```diff
@@ -260,6 +260,23 @@ def test_remove(self):
                 'TestCommentModeration.test_remove'):
             self.reddit.comment('da2g5y6').mod.remove(spam=True)
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_send_removal_message(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestCommentModeration.test_send_removal_message'):
+            comment = self.reddit.comment('edu698v')
+            mod = comment.mod
+            mod.remove()
```

Comment:
What happens if the submission isn't already removed? Does adding a removal message explicitly remove the message?

Reply:
Trying to send a removal message on an object that's not removed results in a HTTP 400.